src/__tests__/queryByApi.test.js

```diff
@@ -3,56 +3,73 @@ import React from 'react';
 import { Text, Image } from 'react-native';
 import { render } from '..';
 
-test('queryByText nested <Image> in <Text> at start', () => {
-  expect(
-    render(
-      <Text>
-        <Image source={{}} />
-        Hello
-      </Text>
-    ).queryByText('Hello')
-  ).toBeTruthy();
-});
+// test('queryByText nested <Image> in <Text> at start', () => {
```

Comment:
FYI: if you want to skip all other tests than your current one, you can use `test.only` on that test to focus only there :)

Reply:
oops, thanks😅